templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -29,22 +29,10 @@
    "metadata": {},
```

Comment:
<p>Do we not need the sys.path.append because of how the repo is structured now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
We never needed it since toffy was pip installable. I believe it was there just to access the modules in 'toffy'.

Reply:
Right, if that line was missing then it would say unknown package toffy or something like that. That's no longer the case?

Reply:
Yeah that's correct, since the Jupyter kernel would fetch the version of toffy installed in the python package cache.